docs/DevGuide/Parallelization.md

```diff
@@ -100,13 +100,19 @@ making. Here is an example of a `determine_next_phase` function and the `Phase`
 enum class:
 \snippet Test_AlgorithmCore.cpp determine_next_phase_example
 
-In contrast, an evolution executable might have phases `%Initialization`,
-`SetInitialData`, `Evolve`, and `Exit`, but have a similar `switch` or `if-else`
-logic in the `determine_next_phase` function. The first phase that is entered is
-always `%Initialization`. During the `%Initialization` phase the `initialize`
-function is called on all parallel components. Once all parallel components'
-`initialize` function is complete, the next phase is determined and the
-`execute_next_phase` function is called after on all the parallel components.
+In contrast, an evolution executable might have phases
+`%Initialization`, `SetInitialData`, `Evolve`, and `Exit`, but have a
```

Comment:
Looks like we don't need the `%` before `Initialization` anymore.

Reply:
For my own understanding, what does the `%` do?

Reply:
It tells Doxygen not to generate an HTML link to the entity with the name